app/lib/services/voip/MediaCallEvents.ios.test.ts

```
@@ -1,16 +1,70 @@
+					}
+					const idx = list.indexOf(listener);
+					if (idx >= 0) {
+						list.splice(idx, 1);
+					}
+				}
+			};
+		}
+	},
+	NativeEventEmitter: class {
+		// eslint-disable-next-line @typescript-eslint/no-useless-constructor, @typescript-eslint/no-unused-vars
+		constructor(_nativeModule?: unknown) {}
```

ESLint

[error] Unexpected empty constructor.

(no-empty-function)